docs/glossary.md

```diff
@@ -0,0 +1,88 @@
+
+### Apollo Federation
+
+An implementation of **Subschema Directive-based Type Merging** that is defined by the Apollo Federation Specification. GraphQL schemas (**Apollo Federation Subgraph**s) are composed of a single GraphQL schema (**Apollo Federated Graph**) on a Gateway level (**Apollo Federation Gateway**).
+
+### “The Graph”
+
+Branding used by Apollo for describing all their libraries and products.
+
+> [...] the graph is a unified representation of all your data, services, and digital capabilities. The graph enables app developers to delete thousands of lines of boilerplate code, move fast without waiting on backend teams, and keep features consistent across web and mobile platforms
+
+Source: [https://www.apollographql.com/](https://www.apollographql.com/) (2022/02/24)
+> 
+
+### ****Apollo Graph Platform****
+
+Branding used by Apollo for describing their open-source products.
+
+> ****The Apollo Graph Platform****
+The leading open-source GraphQL implementation with 17M downloads per month and the only end-to-end cloud management solution for GraphQL.
+
+Source: [https://www.apollographql.com/](https://www.apollographql.com/) (2022/02/24)
+> 
+
+### Apollo Federated Graph
+
+An implementation of Type Merging. The result of combining multiple Apollo federation Subgraphs is an Apollo Federated Graph. The Apollo Federated Graph is usually exposed via an Apollo Federation Gateway or Apollo Federation compatible gateway (e.g. graphql-mesh).
+
+### Apollo Federation Subgraph ([Link](https://www.apollographql.com/docs/federation/subgraphs/))
+
+A service that exposes a GraphQL schema that is consumed by a gateway in order to create a federated graph. The services schema uses directive annotations on types and fields as a hint for a gateway on how it should combine the services schema with the schema of the other Apollo federation subgraphs. A Apollo Federation Subgraph is a GraphQL schema with Apollo Federation schema directives. Also See **Subschema**.
+
+### Apollo Federated Graph ([Link](https://www.apollographql.com/docs/federation/))
+
+The process of composing multiple Apollo Subgraphs into one schema, usually via a Gateway.
+
+### Apollo Studio
+
+A schema registry for registering and validating Apollo federation subgraphs.
+
+### Apollo Federation Gateway
+
+A gateway that usually uses a schema registry (Apollo Studio) for fetching the apollo subgraph configurations. It is the entry point for incoming GraphQL requests. It analyzes the incoming operations, builds a query plan in order to find the quickest way of retrieving all the data from the apollo federation subgraphs, and then executes that plan.
+
+The approach of combining multiple GraphQL services into a unified GraphQL Graph on a Gateway using meta-information applied on the individual GraphQL services schema definitions.
+
```

Comment:
Thanks for putting this together @n1ru4l 👏 

I'm not 100% sure if this is covered by an existing glossary item. I think it's unique in that is isn't specifying directives and only requires the `id: ID!` field is implemented via interface.

The spec for "global object identification" lends itself well to schema merging for my team's needs. This solved many of our schema composition problems in a very simple way, i.e. subschemas did not need to go out of their way to opt-in to composition.
```suggestion

### Node interface

The [Global Object Identification][] [spec][relay-goi-spec] defines a `Node` interface, which enables composition if shared types implement it.

[Nautilus Gateway](https://github.com/nautilus/gateway) is one such implementation. It uses these [merge strategies](https://github.com/nautilus/gateway/blob/master/docs/mergingStrategies.md) and one interface type to compose subschemas.

[Global Object Identification]: https://graphql.org/learn/global-object-identification/
[relay-goi-spec]: (https://relay.dev/graphql/objectidentification.htm)
```

Reply:
done via https://github.com/graphql/composite-schemas-wg/pull/34/commits/a511f6fc2469ba8a6f68b623aedf5239e2da84db